.github/workflows/R-CMD-check.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       R_KEEP_PKG_SOURCE: yes
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
I noticed that these checkout actions are out of date, so I took a minute to update them. I didn't look at the rest of the actions particularly thoroughly in the interest of time.

Reply:
Nah the rest are fine and had their versions bumped relatively recently.